services/home/src/components/JoinStudy/StudyTitle/style.ts

```diff
@@ -0,0 +1,8 @@
+import styled from '@emotion/styled';
+
+export const Title = styled.div`
+  font-weight: 700;
+  font-size: 32px;
+  color: #ffffff;
+  margin-bottom: 24px;
```

Comment:
```suggestion
  margin-bottom: 24px; 
```

margin은 밑으로 주지말고 위로 주는게 추후 개선에도 편해요! 하단 박스가 `margin-top`을 주는게 좋다는 의견입니당 👍